src/test/java/org/jitsi/videobridge/health/HealthIntervalPropertyTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
```

Comment:
This is a real quibble but brand new files (with no content copied from elsewhere) should probably say 2019 - present.

Reply:
i haven't updated my template because i've been too lazy to verify that it can be done in a way that won't accidentally update old files as well.   i hope you'll forgive the transgression of leaving this as-is, as without the template i'll have to apply the new behavior manually and therefore inconsistently.